strategy/payload_transformer_protobuf.go

```diff
@@ -0,0 +1,112 @@
+package strategy
```

Comment:
Can you please move this to a protobuf package because we don't want every other project using GoEngine to require Protobuf.

Reply:
Huh, why do I only see this now? Sure will. :)

Reply:
Please see latest commit to check if I didn't overthink this.